docs/src/main/asciidoc/getting-started-testing.adoc

```diff
@@ -980,7 +980,82 @@ If you are using the `quarkus-hibernate-orm-panache` or `quarkus-mongodb-panache
 
 == Testing Security
 
-If you are using Quarkus Security, check out the link:security-testing[Testing Security] section for information on how to easily test security features of the application.
+Quarkus provides explicit support for testing with different users, and with the security subsystem disabled. To use
+this you must include the `quarkus-test-security` artifact:
+
+[source,xml]
+----
+<dependency>
+    <groupId>io.quarkus</groupId>
+    <artifactId>quarkus-test-security</artifactId>
+    <scope>test</scope>
+</dependency>
+----
+
+This artifact provides the `io.quarkus.test.security.TestSecurity` annotation, that can be applied to test methods and
+test classes to control the security context that the test is run with. This allows you to do two things, you can disable
+authorization so tests can access secured endpoints without needing to be authenticated, and you can specify the identity
+that you want the tests to run under.
+
+A test that runs with authorization disabled can just set the enabled property to false:
+
+[source,java]
+----
+@Test
+@TestSecurity(authorizationEnabled = false)
+void someTestMethod() {
+...
+}
+----
+
+This will disable all access checks, which allows the test to access secured endpoints without needing to authenticate.
+
+You can also use this to configure the current user that the test will run as:
+
+[source,java]
+----
+@Test
+@TestSecurity(user = "testUser", roles = {"admin", "user"})
+void someTestMethod() {
+...
+}
+----
+
+This will run the test with an identity with the given username and roles. Note that these can be combined, so you can
+disable authorization while also providing an identity to run the test under, which can be useful if the endpoint expects an
+identity to be present.
+
+[WARNING]
+====
+The feature is only available for `@QuarkusTest` and will **not** work on a `@NativeImageTest`.
+====
+
+=== OpenId Connect Bearer Token Integration Testing
```

Comment:
@stuartwdouglas 

Adding the links to OIDC/JWT test sections is good though I'm not sure about the structure. Here different testing concepts are introduced, `@TestSecurity` (in its simplest form which works for all the security extensions), `Wiremock`.
While the OIDC/JWT sections cover `@TestSecurity` extensions and Wiremock and more. with OIDC/JWT sections linking back to this section. So the added sections change the granulariry
But I agree the one way linking is not ideal, it should be 2 way